### PR TITLE
feat: warn if missing code coverage information

### LINKED ISSUE #117: Ineffective configuration should error
When I successfully run tests while trying to set up code coverage I can see a report such as this:

```
----------|---------|----------|---------|---------|-------------------
File      | % Stmts | % Branch | % Funcs | % Lines | Uncovered Line #s 
----------|---------|----------|---------|---------|-------------------
All files |       0 |        0 |       0 |       0 |                   
----------|---------|----------|---------|---------|-------------------
```

I clearly failed to properly configure my coverage collection but there is no indication of that failure.

These files are included in plugins:
```
  on('task', require('@cypress/code-coverage/task'))
  on('file:preprocessor', require('@cypress/code-coverage/use-babelrc'))
```
This is in support:
```
import '@cypress/code-coverage/support'
```

Could any of these tools throw if they were used but coverage was never processed?

### PR DESCRIPTION
**Pull request by bahmutov**
- closes #117 

Gives a warning if cannot find neither e2e nor unit test code coverage. The warning has a link that opens on click right inside the test runner's browser, goes to https://github.com/cypress-io/code-coverage#instrument-your-application

Screen Shot 2020-02-04 at 10 38 47 AM

- if there is no e2e code coverage, but at least unit test coverage gives a "note"

Screen Shot 2020-02-04 at 10 40 32 AM

### CODE REVIEW

File: support.js [hunk 1]
```diff
@@ -53,9 +57,26 @@ if (Cypress.env('coverage') === false) {
     windowCoverageObjects.forEach(cover => {
       sendCoverage(cover.coverage, cover.pathname)
     })
+
+    if (!hasE2ECoverage()) {
+      if (hasUnitTestCoverage()) {
+        cy.log(`👉 Only found unit test code coverage.`)
```
Comment (jennifer-shehane) [inline support.js]: I think we should make these logs more specific to show that this logging is coming from the code-coverage plugin. People put a LOT of stuff in their repos and I'm afraid they'll see this random log and assume it's coming directly from Cypress somewhere. Thoughts to make this clearer:

- Have this be like, a custom command that logs the command name as 'CODE COVERAGE'? 
- At the end of log say something like 'logged from [code-coverage](https://github.com/cypress-io/code-coverage)

**Comment by bahmutov** [comment]
Good idea, will add note

**Comment by bahmutov** [comment]
I have added the same test to every message coming from this plugin

Screen Shot 2020-02-04 at 2 12 58 PM

**Pull request merged**